### DISCUSSION TITLE
max value for bar chart Y axis

### DISCUSSION THREAD

**Discussion by mwmcode** [recharts/recharts#2731] [Q&A]
hello,

is there a way to set max value for bar charts Y axis?

For example, max value for Y axis is 3, and data set is `{value: 1, name: 'a'}, {value: 1, name: 'b'}]`

<img width="213" alt="Screen Shot 2021-12-07 at 3 43 49 pm" src="https://user-images.githubusercontent.com/13101565/145108395-77f9df91-2dce-445e-b403-b8c3206adf33.png">

**Comment by ufdeveloper** [comment]
I have a similar question for min value. I'm plotting running heart rates and all the values hover around the top because Y axis starts at 0.

**Comment by ufdeveloper** [comment]
I think I just figured it out. You need to use the domain property of YAxis with ['auto', 'auto']. The default is [0, 'auto']
